app/components/title-input.js

```diff
@@ -0,0 +1,22 @@
+import Ember from 'ember';
```

Comment:
one last question: cant't the behaviour this provides also be done with CSS `:focus`?